### ISSUE TITLE
[COGNITION]: Does the visual representation of STEM question responses belong on the application review screen?

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#6545]
## Description

**❗️ Must** - review this issue for design best practice on the review screen

The set of visual questions and icons we're using on step 2 to determine if a user is a likely candidate for the STEM scholarship are being used again in the review application step. It seems a bit confusing here, and these might be better served as question and answer rows like the questions above them. Screenshot attached of the item that could be considered for redesign.

/cc @emilywaggoner && @CrystabelReiter 

## Point of Contact

**VFS Point of Contact:** _Trevor, Emily, Crystabel_

## Acceptance Criteria

**Definition of done:**
* [ ] Review application screen is reviewed for design consistency
* [ ] Any code and layout changes are reviewed in staging for design, accessibility passes

## Screenshots or Trace Logs

![Screen Shot 2020-03-04 at 3.57.00 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/06c2f3f4-e06b-4d04-8b70-b5cef3dc7d2d)

**Comment by 1Copenut** [comment]
This issue is a high priority item, because it makes the review process a bit confusing. It's not a launch blocker, but should be prioritized to be reviewed and adjusted quickly after launch.

**Comment by 1Copenut** [comment]
@desireeturner 
This has been updated and verified in Staging. I'll move it to closed.
